books/projects/aleobft/dynamic/anchors.lisp

```diff
@@ -27,8 +27,12 @@
   :long
   (xdoc::topstring
    (xdoc::p
-    "An anchor is a certificate in a DAG
-     that is authored by the leader for the round of the certificate.")
+    "An anchor is a leader certificate that is committed to the blockchain,
+     either directly, by receiving enough votes (see @(see elections)),
+     or because it is on the path from one receiving enough votes to
+     a previously committed anchor.
```

Comment:
And I would delete this line. Reason for the suggested edits: the first time anchors are committed, the path is not to any previously committed anchor, but to "round 0".

Reply:
Added something about round 0.